client/boot/index.js

```diff
@@ -163,13 +163,17 @@ function boot() {
 
 		// Create layout instance with current user prop
 		Layout = require( 'layout' );
-		layout = React.render( React.createElement( Layout, {
```

Comment:
I will get around to that once we decide what to do with global store (do we want singleton?)